### PR TITLE
Country metadata schema

### LINKED ISSUE #45: Add schema for map metadata
store map metadata and add a flag for updating it

### PR DESCRIPTION
**Pull request by TotallyNot**
resolves #45 

- added the `Country` schema to store static information such as country names, coordinates and regions
- added `yargs` for better option parsing
- cleaned up the bot bootstrapping code, and split command updates into a separate script
- added `update_resources` command to update the country data

(The schema isn't actually used by anything yet)

### CODE REVIEW

File: src/entity/Country.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    name!: string;
+
+    @Column({ type: "char", length: 2 })
+    code!: string;
+
+    @Column({ type: "float8" })
+    latitude!: number;
+
+    @Column({ type: "float8" })
+    longitude!: number;
```
Comment (Natty-Boh) [inline src/entity/Country.ts:30]: do we want to maybe also save coastline? we dont do anything with it yet, but its technically part of the static data and maybe if we ever figure out the naval effectiveness bands...

Comment (TotallyNot) [reply src/entity/Country.ts:30]: good point. It isn't affected by nukes, right?

Comment (Natty-Boh) [reply src/entity/Country.ts:30]: uh i actually have no idea, i never considered it

File: src/scripts/updateStaticData.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { makeLogger } from "../logger";
+
+import { Country, Region } from "../entity/Country";
+import { getWorld } from "../wrapper/wrapper";
+
+const logger = makeLogger(module);
+
+export async function updateStaticData() {
+    try {
+        logger.info("Updating static map data...");
```
Comment (Natty-Boh) [inline src/scripts/updateStaticData.ts:12]: since its static seems kind of silly to update it at all.

Comment (TotallyNot) [reply src/scripts/updateStaticData.ts:12]: It needs to be loaded into the db
And if there'll ever be a change to the map it'll need to be updated

**Review by Natty-Boh — APPROVED** [review]

**Pull request merged**